service/policy/db/migrations/20250703000000_obligations.sql

```diff
@@ -0,0 +1,116 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE IF NOT EXISTS obligation_definitions
+(
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    namespace_id UUID NOT NULL REFERENCES attribute_namespaces(id),
+    -- name is a unique identifier for the obligation definition within the namespace
+    name VARCHAR NOT NULL,
+    -- implicit index on unique (namespace_id, name) combo
+    -- index name: obligation_definitions_namespace_id_name_key
+    UNIQUE (namespace_id, name)
+);
+
+CREATE TABLE IF NOT EXISTS obligation_values_standard
+(
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    obligation_definition_id UUID NOT NULL REFERENCES obligation_definitions(id),
+    -- value is a unique identifier for the obligation value within the definition
+    value VARCHAR NOT NULL,
+    -- implicit index on unique (obligation_definition_id, value) combo
+    -- index name: obligation_values_standard_obligation_definition_id_value_key
+    UNIQUE (obligation_definition_id, value)
+);
+
+CREATE TABLE IF NOT EXISTS obligation_triggers
+(
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    attribute_value_id UUID NOT NULL REFERENCES attribute_values(id),
+    obligation_value_id UUID NOT NULL REFERENCES obligation_values_standard(id),
```

Comment:
Can you please handle all the cascade deletion cases? For example, deleting these objects should cascade down to referencing rows:
- attribute values
- obligation values
- obligation definitions
- namespace
- actions

Reply:
done